py-polars/polars/expr/datetime.py

```diff
@@ -1966,29 +1967,24 @@ def offset_by(self, by: str) -> Expr:
         │ 2006-01-01 00:00:00 ┆ 2003-11-01 00:00:00 │
         └─────────────────────┴─────────────────────┘
 
-        To get to the end of each month, combine with `truncate`:
```

Comment:
there's `.dt.month_end()` now, so this example feels redundant - I'm removing it, to put a more relevant example of `offset_by` instead